typescript-conditionals/main.ts

```diff
@@ -0,0 +1,130 @@
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+/*  */
+function isUnderFive(number: number): boolean {
+  const isLessThanFive: boolean = number < 5;
+  return isLessThanFive;
+}
+/* The log method of the console object is being called by two arguments, a string
+& the numbers being called by the variable isUnderFive. */
+console.log('Is 4 < 5?:', isUnderFive(4));
+console.log('Is 9 < 5?:', isUnderFive(9));
+
+/*  */
+function isEven(number: number): boolean {
+  const isNumberEven: boolean = number % 2 === 0;
+  return isNumberEven;
+}
+/* The log method of the console object is being called by two arguments, a string
+& the numbers being called by the variable isEven. */
+console.log('Is 4 divisible by 2?:', isEven(4));
+console.log('Is 9 divisible by 2?:', isEven(9));
+
+/*  */
+function startsWithJ(string: string): boolean {
+  const characterStartWithJ: boolean = string[0] === 'J';
+  return characterStartWithJ;
+}
+/* The log method of the console object is being called by two arguments, a string
+& the strings being called by the variable startsWithJ. */
+console.log('Does JavaScript start with J?:', startsWithJ('JavaScript'));
+console.log('Does CSS start with J?:', startsWithJ('CSS'));
+
+/*  */
+interface Person {
+  name?: string;
+  age: number;
+}
+/*  */
+function isOldEnoughToDrink(person: Person): boolean {
+  const isLegalToDrink: boolean = person.age >= 21;
+  return isLegalToDrink;
+}
+/* The commands are in the console */
+
+/*  */
+function isOldEnoughToDrive(person: Person): boolean {
+  const isLegalToDrive: boolean = person.age >= 16;
+  return isLegalToDrive;
+} /* The commands are in the console */
+
+/*  */
+function isOldEnoughToDrinkAndDrive(person: Person): boolean {
+  const isLegalToDrink: boolean = person.age >= 21;
+  const isLegalToDrive: boolean = person.age >= 16;
+  if (isLegalToDrink && isLegalToDrive) {
+    return false;
+  }
+  return false;
+}
+
+/*
+There is an if statement with a conditional expression checking if ph is less than 0 or pH > 14 and an opening curly brace for the code block. */
+function categorizeAcidity(pH: number): string {
+  if (pH < 0 || pH > 14) {
+    return 'Invalid pH level';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7) {
+    return 'base';
+  } else {
+    return 'acid';
+  }
+}
+/* The log method of the console object is being called by two arguments, a string &
+the number being called by the variable categorizeAcidity. */
+console.log('acidic level:', categorizeAcidity(12));
+
+/*
+There is a switch statement with a conditional expression evaluating the variable name.
+There is a case clause checking for the strings yakko, wakko, and dot. */
+function introduceWarnerBro(name: string): string {
+  switch (name.toLowerCase()) {
+    case 'yakko':
+    case 'wakko':
+      return "We're the warner brothers!";
+    case 'dot':
+      return "I'm cute";
+    default:
+      return 'Goodnight everybody!';
+  }
+}
```

Comment:
Nice job implementing a fall through! One of the things I was going to touch on in Q&A!